nyc_translate_setup/serge/data/configs/testconfig.serge

```diff
@@ -0,0 +1,502 @@
+            */
+            debug                               YES
+        }
+    }
+
+    # connection parameters for a remote source control repository
+    vcs
+    {
+        # (STRING) VCS plugin name
+        plugin                      git
+
+        # [OPTIONAL] Plugin data. Each plugin may have specific parameters
+        # inside the `data` block. See the documentation for each specific
+        # plugin for more information.
+        data
+        {
+            # (STRING) Absolute path to local folder where local checkout will be stored.
+            local_path              /var/serge/data/vcs
+
+            /*
+            As you can see from the line above, one can use `%ENV:SOMEVAR%` macro
+            anywhere in the string — it will be substituted with the value
+            of the `SOMEVAR` environment variable; if the variable with that name
+            is not defined, the script will fail with the corresponding message.
+            This allows to use the same configuration file in different environments
+            (e.g. different developers' and build machines) or to reuse
+            common configuration parameters across multiple configuration files.
+            */
+
+            # (STRING) Path to a single remote repository
+            # to sync with local `data_dir` folder
+            remote_path             git@github.com:as1729/random-json-content.git
+            # --- OR ---
+            # (MAP) A key-value list of local subfolders to create and their
+            # corresponding remote repositories (if the localizable data for the
+            # single localization project is located in several per-component
+            # or per-library repositories)
+            # remote_path
+            # {
+            #     # one can specify branch name after the '#'.
+            #     # below, the `v5` branch us used
+            #     main                ssh://l10n@git.example.com/myapp#v5
+            #     # if no branch is specified, `master` branch is used by default
+            #     widget              ssh://l10n@git.example.com/myappwidget
+            # }
+
+            # (BOOLEAN) [OPTIONAL] should the newly generated
+            # files be added to the remote repository automatically?
+            # (YES or NO, defaults to NO)
+            add_unversioned         YES
+
+            # (STRING) [OPTIONAL] Commit message
+            # Default: 'Automatic commit of updated project files'
+            commit_message          Automatic commit of updated project files
+
+            # (STRING) public committer name
+            name                    L10N Robot
+
+            # (STRING) committer's email address
+            email                   l10n-robot@example.com
+        }
+    }
+}
+jobs
+{
+    # First job definition
+    {
+        # (STRING) Unique job identifier string.
+        # Format is arbitrary
+        id                          nyc.test.job
```

Comment:
Will we want a wrapper script that takes care of chores like generating a unique job id?